README.md

```diff
@@ -8,42 +14,59 @@
 <img src="https://raw.githubusercontent.com/haadcode/orbit/master/screenshots/screenshot7%202016-09-02.png" width="50%">
 <img src="https://raw.githubusercontent.com/haadcode/orbit/master/screenshots/screenshot6%202016-04-17.png" width="50%">
 
+## Table of Contents
+
+- [Project Status](#project-status)
+- [Run](#run)
+  - [Live Demo](#live-demo)
+  - [Desktop](#desktop)
+  - [Browser](#browser)
+- [Development](#development)
+  - [Requirements](#requirements)
+  - [Get the source code](#get-the-source-code)
+  - [Install dependencies](#install-dependencies)
+  - [Run Tests](#run-tests)
+  - [Browser app development](#browser-app-development)
+    - [Build](#build)
+  - [Desktop app development](#desktop-app-development)
+    - [Build](#build-1)
+- [Contributing](#contributing)
+- [License](#license)
+
```

Comment:
I disagree; I think this is a very useful way to see the structure of the document, at a glance. You can't assume that new readers will have time to read everything, or that you will know what they are directly interested in; this helps guide them through the document. 

But I'll remove it, because this, among a few other disagreements, is I think your main reason for not merging this sooner.